scripts/skills/racial/rf_zombie_racial.nut

```diff
@@ -0,0 +1,99 @@
+this.rf_zombie_racial <- ::inherit("scripts/skills/skill", {
+	m = {
+		FatigueEffectMult = 0.0,
+		FatigueDealtPerHitBonusMult = 1.0
+	},
+	function create()
+	{
+		this.m.ID = "racial.rf_zombie";
+		this.m.Name = "Wiederganger";
+		this.m.Description = "This character is a wiederganger.";
+		this.m.Icon = "ui/orientation/zombie_01_orientation.png";
+		this.m.Type = ::Const.SkillType.Racial | ::Const.SkillType.StatusEffect;
+		this.m.Order = ::Const.SkillOrder.Last;
+	}
+
+	function getTooltip()
+	{
+		local ret = this.skill.getTooltip();
+
+		if (this.m.FatigueEffectMult == 0.0)
+		{
+			ret.push({
+				id = 10,
+				type = "text",
+				icon = "ui/icons/fatigue.png",
+				text = ::Reforged.Mod.Tooltips.parseString("Builds no [Fatigue|Concept.Fatigue]")
+			});
+		}
+		else if (this.m.FatigueEffectMult != 1.0)
+		{
+			ret.push({
+				id = 10,
+				type = "text",
+				icon = "ui/icons/fatigue.png",
+				text = ::Reforged.Mod.Tooltips.parseString("Build up " + ::MSU.Text.colorizeMult(this.m.FatigueEffectMult, {InvertColor = true}) + " less [Fatigue|Concept.Fatigue]")
+			});
+		}
+
+		if (this.m.FatigueDealtPerHitBonusMult != 0.0)
+		{
+			local bonusFatiguePerHit = ::Const.Combat.FatigueReceivedPerHit * this.m.FatigueDealtPerHitBonusMult;
```

Comment:
Should add a check for this `bonusFatiguePerHit` being not 0. Because a mod may change `::Const.Combat.FatigueReceivedPerHit` to 0 for example.